docs/apv.md

```diff
@@ -348,9 +348,12 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 ### Artikel 34 - Roleplay & Ledenaantal Criminele Organisaties
 
 1. Criminele organisaties mogen een maximaal ledenaantal hebben van 25 personen, hierin is een vrije keuze voor het aantal leden met de 'loopjongens' rang (onder loopjongens word gezien officiële leden die zich nog moeten bewijzen en geen toegang hebben tot het F6 menu).
-2. Een roleplayscenario van/met een criminele organisatie mag maximaal bestaan uit 20 personen, inclusief de leden van de criminele organisatie zelf. Dit houdt dus in dat iedereen die op dat moment enige medewerking verleent aan de criminele organisatie, ook indien zij officieel niet tot de criminele organisatie behoren, tot dit aantal wordt gerekend.
-3. Overtreding van het ledenaantal genoemd in lid 1 & 2 zal een straf worden uitgedeeld volgens de 1e categorie aan de eigena(a)r(en) & deelnemers van de organisatie.
-4. Bij herhaalde overtreding zal er een straf van de 2e categorie volgen.
+2. Bij een overtreding van het ledenaantal genoemd in lid 1 zal een straf worden uitgedeeld volgens de 1e categorie aan de eigena(a)r(en) & deelnemers van de organisatie.
+3. Bij herhaalde overtreding zal er een straf van de 2e categorie volgen.
+4. Criminele organisaties mogen met een maximum aantal van 20 personen een roleplayscenario houden, dit mag met hun volledige organisatie of met leden van andere Officiële of onofficiële gangs of personen, mits dit een geldige en logische reden heeft in de roleplay.
```

Comment:
er mogen 25 man in je gang zitten maar je mag maar met 20 man een rp aangaan? wat als je een feest geeft met al je lenden?

Reply:
"Criminele roleplay" beter dan?

Reply:
Dan zou je nog niet met heel je gang wat neer mogen zetten?